src/Utils/BlockTemplateUtils.php

```diff
@@ -39,6 +39,16 @@ class BlockTemplateUtils {
 	 */
 	const PLUGIN_SLUG = 'woocommerce/woocommerce';
```

Comment:
can we pull it from `BlockTemplateUtils::PLUGIN_SLUG` instead of redeclaring locally as well like you did with `DEPRECATED_PLUGIN_SLUG`?